charts/tigera-operator/templates/tigera-operator/02-tigera-operator.yaml

```diff
@@ -108,7 +108,7 @@ spec:
                 optional: true
         # Install any v3 API resources provided in the calico-resources ConfigMap.
         - name: create-initial-resources
-          image: {{.Values.calicoctl.image}}:{{.Values.calicoctl.tag}}
+          image: {{.Values.calicoctl.registry}}/{{.Values.calicoctl.image}}:{{.Values.calicoctl.tag}}
```

Comment:
This will need to be reverted as well in order to avoid breaking users on upgrade.